libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -17,6 +17,13 @@
 #define AP_HEATER_IMU_INSTANCE 0
 #endif
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
```

Comment:
There is a `SENSOR_RATE_DEBUG` define, although it seems not to be used anywhere at the moment. Looks like these dubugs are also not used anywhere in this patch? They should be behind a debugging define of some sort.

Reply:
yeah I will take them out from this PR

Reply:
Fixed